### PR TITLE
CPT 11 new tools

### PR DESCRIPTION
**Pull request by jasonjgill**
New PR for addition of 11 new CPT tools to usegalaxy.eu

### CODE REVIEW

**Comment by jasonjgill** [comment]
Hi @bgruening, is there anything else I should be doing with this PR? Is it pointing at the right place?

**Comment by jasonjgill** [comment]
Hi @bgruening, is there anything else I should be doing with this PR? Is it pointing at the right place?

**Review by hexylena — APPROVED** [review]
At least has my stamp of approval, for what it's worth as an ex-eu person :)

**Comment by bgruening** [comment]
@jasonjgill sorry, I was traveling a lot and wanted to look at your tools.

I have created an issue with a few things that I have found: https://github.com/TAMU-CPT/CPT-ToolshedSource/issues/1 and also a example PR with a few chances: https://github.com/TAMU-CPT/CPT-ToolshedSource/pull/2

Why have you chosen to put the tools into the GalaxyP yaml file? I create https://github.com/TAMU-CPT/CPT-ToolshedSource/pull/2 which will add your own YML file, where we can collect all your tools.

Does this sound like a plan? Do you need any help?

**Comment by hexylena** [comment]
`planemo lint` should be catching some of these things, you should be running that on a github action to catch many issues ahead of time.

**Comment by jasonjgill** [comment]
@bgruening @hexylena Sorry I thought that adding to the GalaxyP yaml was they way to get them added to usegalaxy.eu? I thought we would go for adding to usegalaxy first then work on making them compliant to IUC, we can go the route of adding to IUC first but that will take longer, for reasons you have already pointed out.

Thanks for the info provided in the PR you made in the CPT toolshed. If we are going to IUC we can poke away at these changes to make them compliant.

**Comment by hexylena** [comment]
They could probably go in their own "phage.yml" or "genome-annotation.yaml"

**Comment by jasonjgill** [comment]
Can do, would that just be a new file in the same usegalaxy-eu-tools/ directory?

**Comment by jasonjgill** [comment]
@bgruening trying to get this moving again...can these be merged or did you want the tools to go into a separate .yaml file as @hexylena suggested?

**Comment by jasonjgill** [comment]
@bgruening per @hexylena suggestion I made a commit to create a new file phage.yaml but not sure if this will replace edits to the tools_galaxyp.yaml...

File: tools_galaxyp.yaml [hunk 1]
```diff
@@ -1036,3 +1036,47 @@ tools:
...
+    owner: cpt
+    tool_panel_section_label: FASTA/FASTQ
+    
+  - name: cpt_get_orfs
+    owner: cpt
+    tool_panel_section_label: Annotation
+    
+  - name: cpt_convert_mga
+    owner: cpt
+    tool_panel_section_label: Convert Formats
```
Comment (hexylena) [inline tools_galaxyp.yaml:1050]: `galaxyp` is specifically proteomics, I don't think they should be in this file at all.

**Comment by hexylena** [comment]
> I thought that adding to the GalaxyP yaml was they way to get them added to usegalaxy.eu?

just to clarify, ALL files in this repository are in usegalaxy.eu, `galaxyp` stands for `galaxy proteomics`.

I've resolved the merge conflict, by removing the changes to `galaxyp`, and just leaving `phage.yaml`

**Comment by jasonjgill** [comment]
Thanks! That's my bad, I thought GalaxyP was Galaxy Project...

**Comment by hexylena** [comment]
> GalaxyP was Galaxy Project

I can see how that would be confusing :sweat_smile: I probably would've made the same mistake.

**Comment by bgruening** [comment]
I have merged: https://github.com/usegalaxy-eu/usegalaxy-eu-tools/pull/523

**Comment by bgruening** [comment]
We have installed those I think. Please reopen if I'm wrong.

**Pull request closed without merging**